### PR TITLE
#5461728 Bug fix for get-serviceprovider

### PR DESCRIPTION
**Pull request by dihan0604**

### CODE REVIEW

**Comment by DeepakRajendranMsft** [comment]
Lets enable the test for this please.

File: src/ResourceManagement/Network/Microsoft.Azure.Management.Network/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -20,7 +20,7 @@
 [assembly: AssemblyDescription("Provides Microsoft Azure Network management functions for managing the Microsoft Azure Network service.")]
 
 [assembly: AssemblyVersion("3.0.0.0")]
-[assembly: AssemblyFileVersion("3.0.5.0")]
```
Comment (DeepakRajendranMsft) [inline src/ResourceManagement/Network/Microsoft.Azure.Management.Network/Properties/AssemblyInfo.cs]: why not 3.0.6-preview?

**Comment by dihan0604** [comment]
I have several private package which already used 3.0.6, to test this change with powershell, i was using 3.0.8

**Comment by dihan0604** [comment]
I have tested it with powershell, I'd like to enable test however i am not familiar with it. Could I re-enable this test later after this release?

**Pull request merged**